examples/terraform-aws-ssm-example/main.tf

```diff
@@ -0,0 +1,52 @@
+provider "aws" {
```

Comment:
Please see the other `main.tf` files for the typical comment structure we use in these files to help make them more readable.